backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/TimeMachineService.java

```diff
@@ -51,6 +56,7 @@ public Norm applyPassiveModifications(ApplyPassiveModificationsUseCase.Query que
     } catch (final MandatoryNodeNotFoundException e) {
       return norm;
```

Comment:
Maybe add a log here to inform that norm with eli X did not contain a akn:meta block?